### PR TITLE
Add annotation check to exoplanets feedback

### PR DESCRIPTION
**Pull request by eatyourgreens**
Describe your changes.
Adds an annotation check so that SGL exoplanets feedback only shows depending on the answer to the question.

# Review Checklist

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [ ] Did you deploy a staging branch?
- [ ] Did you get any type checking errors from [Babel Typecheck](https://github.com/codemix/babel-plugin-typecheck)

## Optional

- [ ] If it's a new component, is it in ES6? Is it clear of warnings from ESLint?
- [ ] Have you replaced any `ChangeListener` or `PromiseRenderer` components with code that updates component state?
- [ ] If changes are made to the classifier, does the dev classifier still work?
- [ ] Have you added in [flow type annotations](https://flowtype.org/docs/type-annotations.html)?

### CODE REVIEW

File: app/classifier/classification-summary.jsx [hunk 1]
```diff
@@ -36,6 +36,14 @@ class ClassificationSummary extends React.Component {
     return false;
   }
 
+  showExoplanetSimFeedback() {
+    return (this.props.workflow.configuration &&
+      this.props.workflow.configuration.sim_notification &&
+      this.isSubjectASim() &&
+      this.props.classification.annotations[0].value === 0
```
Comment (eatyourgreens) [inline app/classifier/classification-summary.jsx]: That's a very good question. I think the exoplanets project requires an answer before you can classify.

Comment (shaunanoordin) [inline app/classifier/classification-summary.jsx]: In that case, I'm OK to leave this as is. There's no logical way to reach this section of the code unless the project is Exoplanets Explorer, and if EE always enforces an answer, then we won't need to worry about nulls.

**Comment by shaunanoordin** [comment]
I've just checked with Grant and Marten - yeah, apparently we have a flooooood of sims in the Subject sets. That's not related to the front-end at all, so I have no further issues to investigate. This PR is LGTM.

**Comment by srallen** [comment]
I updated the tester project I made on staging so that it works with this now. The reason it didn't work is that I had a multiple choice task type setup and now that the annotation is being checked, the task type needs to match what is expected.

**Pull request merged**